src/main/java/de/dennisguse/opentracks/chart/ChartView.java

```diff
@@ -527,6 +542,10 @@ public void computeScroll() {
     public boolean onTouchEvent(MotionEvent event) {
         boolean isZoom = detectorZoom.onTouchEvent(event);
         boolean isScrollTab = detectorScrollFlingTab.onTouchEvent(event);
+
+        // ChartView handles zoom gestures (more than one pointer) and all gestures when zoomed itself
+        requestDisallowInterceptTouchEventInParent(event.getPointerCount() != 1 || zoomLevel != MIN_ZOOM_LEVEL);
```

Comment:
In that way only can swip if there are not zoom.

If you prefer that behaviour, what would you think add double tap to zoom out?

Reply:
Why not :)